pingpong/ud_pingpong.c

```diff
@@ -519,7 +519,11 @@ static int run(void)
 		for (i = 0; i < TEST_CNT; i++) {
 			if (test_size[i].option > opts.size_option)
 				continue;
+
 			opts.transfer_size = test_size[i].size;
+			if (opts.transfer_size > buffer_size)
+				continue;
```

Comment:
Should we print some kind of skipped message / return an error if the user asks for a too-large transfer size?

(ditto for the diff below, too)